src/AnnotatedGroundProgram.cpp

```diff
@@ -448,7 +448,7 @@ void AnnotatedGroundProgram::computeAtomDependencyGraph()
                             const OrdinaryAtom& oatom = ctx->registry()->ogatoms.getByAddress(*en);
                             bool relevant = true;
                             for (int i = 0; i < ea.inputs.size(); ++i) {
-                                if (oatom.tuple[0] == ea.inputs[i] && (!b.isNaf() && prop.isAntimonotonic(i) || b.isNaf() && prop.isMonotonic(i))) {
+                                if (oatom.tuple[0] == (ea.inputs[i] && ((!b.isNaf() && prop.isAntimonotonic(i)) || (b.isNaf() && prop.isMonotonic(i))))) {
```

Comment:
This is definitely an incorrect change, the condition should be
`(oatom.tuple[0] == ea.inputs[i]) && ( (!b.isNaf() && prop.isAntimonotonic(i)) ||( b.isNaf() && prop.isMonotonic(i)) )`.

Actually for readability it would be better to write:
```
const bool antimonotonic =  (!b.isNaf() && prop.isAntimonotonic(i)) || (b.isNaf() && prop.isMonotonic(i));
if ( (oatm.tuple[0] == ea.inputs[i]) && antimonotonic ) {
```